client/X11/xf_floatbar.c

```diff
@@ -401,8 +412,8 @@ static void xf_floatbar_event_expose(xfFloatbar* floatbar)
 	/* draw the host name connected to (limit to maximum file name) */
 	len = strnlen(floatbar->title, MAX_PATH);
 	XSetForeground(display, gc, xf_floatbar_get_color(floatbar, FLOATBAR_COLOR_FOREGROUND));
-	XDrawString(display, floatbar->handle, gc, floatbar->width / 2 - len * 2, 15, floatbar->title,
-	            len);
+	XmbDrawString(display, floatbar->handle, floatbar->fontSet, gc, floatbar->width / 2 - len * 2,
```

Comment:
Can we still use `XDrawString` as a fallback in case font set creation failed (`floatbar->fontSet` is `NULL`)?